docs/docstring_spec.md

```diff
@@ -0,0 +1,60 @@
+# Docstring Spec
+-----------------
+
+`scores` includes a lot of information in its docstrings. These docstrings are used in rendering the readthedocs API documentation, can be built locally as web pages using `sphinx`, can be seen in notebooks and the command-line using the Python's help functionality, or may be read as source code. The tech stack for all of this is somewhat complex. The docstrings include mathematical formulae, references, general links to further reading, coding information (like type information) as well as explanations for users.
+
+This document attempts to set out the contents, layout and syntax which should be adopted where possible, including some information on common issues and gotchas. 
+
+The authors are not specification experts, so this spec is not written in a formal spec language.
+
+For convenience, this spec begins with a template.
+
+# Docstring Template
+--------------------
+
+```
+def function_name(*args, *, **kwargs) -> type hint:
+	'''
+	A short one or two line description of what the function does.
```

Comment:
Depending on the metric, sometimes a paragraph may be a more appropriate length for a description of what the function does. So perhaps it would be worth being a bit less prescriptive about the length here? Maybe something more along the lines of "A brief description of what the function does"?